modules/shared/src/main/scala/org/tessellation/security/signature/SignedValidator.scala

```diff
@@ -37,7 +37,7 @@ trait SignedValidator[F[_]] {
 
   def isSignedExclusivelyBy[A <: AnyRef](
     signed: Signed[A],
-    minSignatureCount: Address
+    signerAddress: Address
```

Comment:
I guess you also need to change the name for the implementation.

Reply:
it was already changed @TheMMaciek